tests/mock_fix_server.py

```diff
@@ -0,0 +1,91 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2019 EdNoepel
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import asyncio
+import os
+import simplefix
+import threading
+
+from concurrent.futures import ProcessPoolExecutor, ThreadPoolExecutor
+
+
+class MockFixServer:
```

Comment:
This implementation should be thrown away.  I considered creating a new one which opens the server side socket and waits, but it'd be expensive.  Buying a solution like _VeriFIX_ is also expensive.  Less discuss again in standup.